plugins/caddy/README.md

```diff
@@ -7,14 +7,14 @@ This is a distributed HTTP cache module for Caddy based on [Souin](https://githu
 
  * Supports most HTTP cache headers defined in [RFC 7234](https://httpwg.org/specs/rfc7234.html) (see the TODO section for limitations)
  * Sets [the `Cache-Status` HTTP Response Header](https://httpwg.org/http-extensions/draft-ietf-httpbis-cache-header.html)
- * REST API to be able to purge the cache and list stored keys.
- * Embedded (or not) distributed cache can be used and a non-distributed one is supported too.
+ * REST API to purge the cache and list stored resources.
+ * Builtin support for distributed cache.
```

Comment:
Are you completely removing the support for YKey and flexible cache providers?

Reply:
I'm working on a fully specified system called `Surrogate-keys` and make a RFC for this.